src/main/java/com/ee417/groupf/model/Order.java

```diff
@@ -0,0 +1,129 @@
+package com.ee417.groupf.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class Order {
+    @JsonProperty("orderId")
+    private long orderId;
+    @JsonProperty("items")
+    private List<Item> items;
```

Comment:
I've merged my code for MenuItems so you'll have to rebase and rename Item to MenuItem as that's the class name but looks good otherwise